src/main/java/Duke.java

```diff
@@ -1,10 +1,172 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
 public class Duke {
+
+    public static final int LENGTH_DEADLINE = 9;
+    public static final int LENGTH_EVENT = 6;
+    public static final int LENGTH_TODO = 5;
+    public static final int SIZE_DONE_COMMAND = 2;
+
+    public static void getDateTime() {
+        LocalDateTime myDateObj = LocalDateTime.now();
+        DateTimeFormatter myFormatObj = DateTimeFormatter.ofPattern("dd-MM-yyyy HH:mm:ss");
+        String formattedDate = myDateObj.format(myFormatObj);
+        System.out.println(formattedDate);
+    }
+
+    public static void completeTask(ArrayList<Task> tasks, int taskIndex) {
+        taskIndex--; // index starts from 0, unlike listing number
+        if ( (taskIndex < tasks.size()) || (taskIndex > 0)) { // check if out of bounce
+            Task currentTask = tasks.get(taskIndex);
+            if (currentTask.getStatus()) { // check if already completed
+                System.out.println("Task already completed!\n");
+            } else {
+                currentTask.markAsDone();
+                System.out.println("Nice! I've marked this task as done:");
+                System.out.println( "["+ currentTask.getTaskType() + "][" + currentTask.getStatusIcon() + "] " + currentTask.getDescription() + "\n");
+            }
+        } else {
+            System.out.println("Error: No such index in use\n");
+        }
+    }
+
+    public static void printList(ArrayList<Task> tasks) {
+        int count = 1;
+        System.out.println("Listing tasks below:");
+
+        if (tasks.isEmpty()) {
+            System.out.println("No tasks at the moment!");
+        } else {
+            for (Task currentTask : tasks) {
+                currentTask.printListDetails(count);
+                count++;
+            }
+        }
+        System.out.println("");
+    }
+
+    public static void printHelp() {
+        System.out.println("Commands: ");
+        System.out.println("List: lists all recorded tasks \nusage: list\n");
+        System.out.println("Done: mark task as completed \nusage: done <task number>\n");
+        System.out.println("Todo: Tasks without date/time \nUsage: todo <task> \n(Avoid using other keywords as the first word)\n");
```

Comment:
You can consider to make this string into constant if you are going to use it for mutiple times.This can improve the readibility of the code.